### PR TITLE
Remove mixer/telemetry-v1 use of request_duration_seconds

### PR DESCRIPTION
**Pull request by jshaughn**
- remove telem-v1 request_duration_seconds metric, v2 supports only request_duration_milliseconds

closes https://github.com/kiali/kiali/issues/3338

### CODE REVIEW

File: graph/telemetry/istio/appender/response_time_test.go [hunk 1]
```diff
@@ -16,12 +16,12 @@ func TestResponseTime(t *testing.T) {
...
-	q0Temp := `histogram_quantile(0.95, sum(rate(istio_request_duration_%s_bucket{reporter="destination",source_workload="unknown",destination_workload_namespace="bookinfo"}[60s])) by (le,source_workload_namespace,source_workload,source_app,source_version,destination_service_namespace,destination_service,destination_service_name,destination_workload_namespace,destination_workload,destination_app,destination_version,response_code,grpc_response_status))`
-	q0 := fmt.Sprintf(`round(((%s > 0) OR ((%s > 0) * 1000.0)),0.001)`, fmt.Sprintf(q0Temp, "milliseconds"), fmt.Sprintf(q0Temp, "seconds"))
+	q0Temp := `histogram_quantile(0.95, sum(rate(istio_request_duration_milliseconds_bucket{reporter="destination",source_workload="unknown",destination_workload_namespace="bookinfo"}[60s])) by (le,source_workload_namespace,source_workload,source_app,source_version,destination_service_namespace,destination_service,destination_service_name,destination_workload_namespace,destination_workload,destination_app,destination_version,response_code,grpc_response_status))`
+	q0 := fmt.Sprintf(`round(%s > 0,0.001)`, q0Temp)
```
Comment (jotak) [inline graph/telemetry/istio/appender/response_time_test.go:20]: is there a typo here? `%s > 0,0.001` ?

Comment (jotak) [reply graph/telemetry/istio/appender/response_time_test.go:20]: ah I get it, it's `round(query, precision)`

File: prometheus/prometheustest/client_test.go [hunk 2]
```diff
@@ -62,15 +62,14 @@ func TestGetServiceMetrics(t *testing.T) {
...
-	mockHistogram(api, "istio_request_duration_seconds", "{"+labels+"}[5m]", 0.35, 0.2, 0.3, 0.8)
 	mockHistogram(api, "istio_request_duration_milliseconds", "{"+labels+"}[5m]", 0.35, 0.2, 0.3, 0.8)
 	mockHistogram(api, "istio_response_bytes", "{"+labels+"}[5m]", 0.35, 0.2, 0.3, 0.9)
 
 	// Test that range and rate interval are changed when needed (namespace bounds)
 	metrics := client.GetMetrics(&q)
 
 	assert.Equal(t, 6, len(metrics.Metrics), "Should have 6 simple metrics")
-	assert.Equal(t, 4, len(metrics.Histograms), "Should have 4 histograms")
+	assert.Equal(t, 3, len(metrics.Histograms), "Should have 4 histograms")
```
Comment (jotak) [inline prometheus/prometheustest/client_test.go:72]: "Should have 3 histograms" then

File: prometheus/prometheustest/client_test.go [hunk 3]
```diff
@@ -131,7 +127,7 @@ func TestGetAppMetrics(t *testing.T) {
...
```
Comment (jotak) [inline prometheus/prometheustest/client_test.go:130]: 3 histograms

File: prometheus/prometheustest/client_test.go [hunk 4]
```diff
@@ -275,7 +268,7 @@ func TestGetNamespaceMetrics(t *testing.T) {
...
```
Comment (jotak) [inline prometheus/prometheustest/client_test.go:271]: 3 histograms

File: graph/telemetry/istio/appender/response_time_test.go (hunk 1 above)
Comment (jshaughn) [reply graph/telemetry/istio/appender/response_time_test.go:20]: No, that looks correct.

**Review by jotak — APPROVED** [review]
LGTM
Tested both graph, side panel, traffic page and metrics page => ok

**Review by mattmahoneyrh — APPROVED** [review]

**Pull request merged**